docs/contributing/style-guide.md

```diff
@@ -1,7 +1,405 @@
+
+## Markdown
+
+This site uses Docusaurus version 2 beta.The 
+[Docusaurus Markdown Features documentation](https://docusaurus.io/docs/markdown-features) 
+is interesting, but is missing descriptions of several key features. Markdown Guide provides a more comprehensive description of available 
+[Basic Syntax](https://www.markdownguide.org/basic-syntax) and 
+[Extended Syntax](https://www.markdownguide.org/extended-syntax/) with some helpful tips for how to use it well.
+Markdown is used for its readability, but for anything where you need more control or more complex styling,
+plain HTML may also be used directly in the file.
+
+## Frontmatter
+
+At the top of each docs page, you need to include these things: 
+
+| Variable | Description |
+| ----------- | ----------- |
+|id|A brief string that uniquely identifies the page within its parent folder. The id and the name of the file should be the same.|
+|title|The main title of the page. This value will automatically be rendered using the H1 style at the top of the page.
+|sidebar_label|This is what will appear in the left hand navigation tree for the page.|
+|description (optional)|This is what appears when the page is referenced in a Google search result.|
+
+### Code Example
+
+```markdown
+---
+id: style-guide
 title: Style Guide Introduction
-sidebar_label: Overview
+sidebar_label: Style Guide
+description: The Sauce Labs Documentation Style Guide
 ---
+```
+
+## Introduction
+
+The first paragraph of the documentation should set the user's expectations for what they will find on the page. 
+Describe the key benefits to the user, but do not include links.
+
+## Headers
+
+For accessibility and SEO reasons, never have an H4 header that isn't under an H3 header, 
+or an H3 header that isn't under an H2 header.
+
+* H1 headers should never be used in a document since the title is automatically generated as an H1.
+* H2 headers are used for SEO, so make sure they succinctly represent what a user will find on the page in that section
+* H3 headers are included in the page's table of contents on the right, so make sure the title describes something
+a user might want to click on
+* H4 headers are to emphasize things within a subsection of the page; these can be longer than the other headers if needed. 
+
+### Code Example
+
+```markdown
+## H2 Header
+### H3 Header
+#### H4 Header
+```
+
+## Content
+
+All words are rendered in the same paragraph even with line breaks, so long as there isn't an empty line.
+As such, it is good practice for each line to be less than 120 characters long for readability, when possible.
+
+### Code Example
+
+```markdown
+This
+will
+all
+be
+on
+one
+line.
+
+The empty line above creates a new paragraph.
+
+This forces a soft return<br />
+rather than creating a new paragraph
+```
+
+### Display
+
+This
+will
+all
+be
+on
+one
+line.
+
+The empty line above creates a new paragraph.
+
+This forces a soft return<br />
+rather than creating a new paragraph
```

Comment:
It might be helpful to have these two sections in a table, so you can see the code and the rendered content side by side.

Reply:
Yeah, I couldn't do this side-by-side, but I got something working vertically.